std/random.d

```diff
@@ -1291,7 +1299,7 @@ random number sequences every run.
 Returns:
 A single unsigned integer seed value, different on each successive call
 */
-@property uint unpredictableSeed() @trusted
+@property uint unpredictableSeed() @trusted nothrow @nogc
```

Comment:
FYI: there is a pending PR and interest to improve the seeding in Phobos. This is usually done by querying the OS APIs which can throw errors, but for me it makes sense to have this function as `nothrow` - especially because we can always fallback to this "naive"  seeding variant in case the improvement ever lands in Phobos.